src/pages/contributing.mdx

```diff
@@ -0,0 +1,293 @@
+# Contributing to the Documentation
+
+This is a non-exhaustive guideline for making contributions to the [NeoForged Documentation][docs] repository. Contributions can be made by forking and cloning the repository and then added via a pull request, or PR, on the [GitHub][docs].
+
+You can run the website locally using [npm]. It is recommended to use a Node Version Manager like [nvm] (Mac, Linux) or [nvs] (Windows) to setup and install npm and Node. From there, you can run the following commands:
+
+```bash
+nvm use # or nvs use on Windows
+npm install
+npm run start
+```
+
+## Principles
+
+This documentation is a guide to help a modder understand and implement a given concept from Minecraft or NeoForged.
+
+This documentation is **not** meant as a tutorial, allowing a modder to copy-paste the examples. If you are looking for a tutorial, there are plenty of videos and pages, which are not linked here, that you can use and follow along with.
+
+This documentation is also **not** meant as documentation for a class. Providing a description of an element is unavoidable when writing a guide; however, if you would like to document a class, you should contribute to [Parchment for Minecraft][parchment] or [NeoForge for NeoForged][neo].
```

Comment:
These two could be notes?

Reply:
This doesn't make as much sense with how the paragraph is structured since it has three not sections. I think, structure-wise it makes sense. Though I will fix that finally.